resolwe/flow/executors/null/run.py

```diff
@@ -19,4 +19,4 @@ class FlowExecutor(BaseFlowExecutor):  # pylint: disable=abstract-method
 
     def run(self, data_id, script, verbosity=1):
         """Do nothing :)."""
-        pass
+        self._send_manager_command('finish')
```

Comment:
As mentioned before I don't like these string commands everywhere as this can be prone to typos and is harder to verify. I suggest these commands be exposed as methods, taking the appropriate parameters and they then internally call `_send_manager_command`.

Reply:
The hardcoded string constant still seems to be here? You probably forgot to replace this with `ExecutorProtocol.FINISH`.